### PR TITLE
Document constraints batch 5

### PR DESCRIPTION
**Pull request by Azdaroth**

### CODE REVIEW

File: content/reference/endpoints/rates_tables.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Rates Tables
 
-1. TOC
-{:toc}
+## Overview
+
+This resource represents Rates Tables - a resource for grouping price-related resources (such as Rates Rules, Seasons, Periods) for [Rentals](/reference/endpoints/rentals/). Rates Tables can be shared between Rentals to make them reusable.
```
Comment (adamgrad) [inline content/reference/endpoints/rates_tables.md:5]:
```suggestion
This resource represents Rates Tables - a resource for grouping price-related resources (such as Rates Rules, Seasons, Periods) for [Rentals](/reference/endpoints/rentals/). Rates Tables can be shared between Rentals to make pricing setup between rentals reusable.
```

Also, there is a bit tricky logic on Core that handled changes to rates table on rental depending on the rates management of that rental (you can check Rental spec from line 548)
But basically if rental has external rates management and a `private` rates table, user will not be able to change this rates table to a different one or update it to a shared table

File: content/reference/endpoints/rates_tables.md [hunk 2]
```diff
@@ -13,25 +14,37 @@
...
+-----------------|---------|--------------------------|
+account          | Integer | Read       | Account's ID related to the Rates Table |
+seasons          | Array   | Read       | Seasons' IDs related to the Rates Table |
+rates_rules      | Array   | Read       | Rates Rules' IDs related to the Rates Table |
+periods          | Array   | Read       | Periods' IDs related to the Rates Table |
+rentals          | Array   | Read       | Rentals' IDs related to the Rates Table |
+-----------------|---------|--------------------------|
+id               | Integer | Read       | Rates Table's ID. |
+name             | String  | Read/Write | Rates Table's name. | **Required**, Maximum length: 50, cannot contain non-printing characters
+shared           | Boolean | Read       | Rates Table's shared (between Rentals) status. (Rates Table is always marked as shared when creating or updating through API) |
```
Comment (adamgrad) [inline content/reference/endpoints/rates_tables.md:43]: Actually, what is the reason for us to enforce shared tables everytime (Even if created as private by user) ? 🤔

File: content/reference/endpoints/rental_agreements.md [hunk 3]
```diff
@@ -3,6 +3,12 @@
 1. TOC
 {:toc}
 
+## Overview
+
+This resource represents Rental Agreements - an agreement that needs to be accepted by [Clients (travelers)](/reference/endpoints/clients/) for [Bookings](/reference/endpoints/bookings/).
+
+The Rental Agreement can be defined on the Account level (a global one that will be applied to all [Rentals](/reference/endpoints/rentals/)), a custom one for Rental (if not defined, it default to Account's one) and on the Booking level (which is the copy of the Rental's one at time of Booking creation).
```
Comment (adamgrad) [inline content/reference/endpoints/rental_agreements.md:10]:
```suggestion
The Rental Agreement can be defined on the Account level (a global one that will be applied to all [Rentals](/reference/endpoints/rentals/)), a custom one for Rental (if not defined, it defaults to Account's one) and on the Booking level (which is the copy of the Rental's one at time of Booking creation).
```

File: content/reference/endpoints/rental_cancelation_policies.md [hunk 4]
```diff
@@ -3,6 +3,12 @@
 1. TOC
 {:toc}
 
+## Overview
+
+This resource represents Rental Cancelation Policies which define the rules and potential penalties for the cancelation of [Bookings](/reference/endpoints/bookings/) (defined either via Rental Cancelation Policy Items or `preset`)
+
+The Rental Cancelation Policy can be defined on the Account level (a global one that will be applied to all [Rentals](/reference/endpoints/rentals/)), a custom one for Rental (if not defined, it default to Account's one) and on the Booking level (which is the copy of the Rental's one at time of Booking creation).
```
Comment (adamgrad) [inline content/reference/endpoints/rental_cancelation_policies.md:10]: default -> defaults

Comment (adamgrad) [inline content/reference/endpoints/rental_cancelation_policies.md:10]: default -> defaults

File: content/reference/endpoints/rentals.md [hunk 5]
```diff
@@ -127,84 +186,132 @@ zip                             | String   | Read       | Rental's zip code.
...
+destination_id                  | Integer  | Write      | Destination id related to the rental. |
+payment_gateway_id              | Integer  | Write      | PaymentGateway id related to the rental. |
+rates_table_id                  | Integer  | Write      | Rates Table id related to the rental. |
+rentals_contact_id              | Integer  | Write      | Contact id related to the rental. |
+rentals_tag_ids                 | Array    | Write      | Tag ids related to the rental. |
+absolute_min_price              | [Decimal](/reference/enums#formats) | Read/Write | Rental's minimum price based on `base_rate_kind`. | Must be greater than or equal to 0 and less than or equal to `base_rate` 
+address1                        | String   | Read/Write | Rental's primary address. |
+address2                        | String   | Read/Write | Rental's optional address. |
+base_rate                       | [Decimal](/reference/enums#formats) | Read/Write | Rental's base rate, used by the rates table to calculate the seasonability and discounted prices. We recommend using the highest price as base rate | Must be greater than 0 if present
```
Comment (adamgrad) [inline content/reference/endpoints/rentals.md:244]: >We recommend using the highest price as base rate

why such recommendation?

File: content/reference/endpoints/rentals.md [hunk 6]
```diff
@@ -127,84 +186,132 @@ zip                             | String   | Read       | Rental's zip code.
...
+certifications                  | [Object](/reference/enums#formats) | Read/Write | Rental's certifications, all allowed values described in [enums section](/reference/enums#rental-certifications). | Must contain valid certifications if present
+management_type                 | String   | Read/Write | Rental's management type. All types are described in [enums section](/reference/enums#rental-management-type) | Must be a valid management type if present
+bookable_on_request             | Boolean  | Read       | Rental's Bookable on request status (requires Rental to be published and Account needs a configuration for contract management) |
+instantly_bookable              | Boolean  | Read       | Rental's instantly bookable status  (requires Rental to be bookable on request, `bookable_online` to be set to `true` and Rental needs to have a Payment Gateway configured) |
+owner_fullname                  | String   | Read/Write | Rental owner's fullname | Maximum length: 100
+owner_email                     | String   | Read/Write | Rental owner's email | Must be avalid email format if present
```
Comment (adamgrad) [inline content/reference/endpoints/rentals.md:297]:
```suggestion
owner_email                     | String   | Read/Write | Rental owner's email | Must be a valid email format if present
```

File: content/reference/endpoints/rates_tables.md (hunk 2 above)
Comment (Azdaroth) [reply content/reference/endpoints/rates_tables.md:43]: I'm not sure really, but that's how it works 😅

File: content/reference/endpoints/rates_tables.md (hunk 1 above)
Comment (Azdaroth) [reply content/reference/endpoints/rates_tables.md:5]: @adamgrad can you suggest a change for it then that would cover all the cases?

File: content/reference/endpoints/rentals.md (hunk 5 above)
Comment (Azdaroth) [reply content/reference/endpoints/rentals.md:244]: @adamgrad not sure but it was there before in the docs, it's not a new recommendation. Would it be better to remove it?

Comment (adamgrad) [reply content/reference/endpoints/rentals.md:244]: @Azdaroth I am quite familiar with the concept of base_rate and I don't quite understand this recommendation, so I believe it's going to be more confusing for the integrating people, so I'd rather remove it.

File: content/reference/endpoints/rates_tables.md (hunk 2 above)
Comment (adamgrad) [reply content/reference/endpoints/rates_tables.md]: ok, then  let's leave it this way for now

**Review by adamgrad — APPROVED** [review]
Awesome 💪

**Pull request merged**